edx_proctoring/templates/proctored_exam/error.html

```diff
@@ -2,20 +2,15 @@
 <div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
   <h3>
     {% blocktrans %}
-      There was a problem with your proctoring session
+      Error with proctored exam
     {% endblocktrans %}
   </h3>
 
-  <h4>
-    {% blocktrans %}
-      Your proctoring session results: <b class="failure"> Unsatisfactory </b>
-    {% endblocktrans %}
-  </h4>
-
   <p>
     {% blocktrans %}
-      Your proctoring session ended before you completed this exam, so your proctoring results are incomplete.
-      You will not be eligible to use this course for academic credit, even if you achieve a passing grade.
+      A technical error has occurred with your proctored exam. To resolve this problem, contact
+      <a href="mailto:{{tech_support_email}}">technical support</a>. You will have to restart the exam.
```

Comment:
@andy-armstrong First 2 sentences are good. After "contact technical support", replace with "All exam data, including answers for completed problems, has been lost. When the problem is resolved you will need to restart the exam and complete all problems again."